docs/templates/index.md

```diff
@@ -69,14 +69,37 @@ coder templates create <template-name>
 > [examples/](https://github.com/coder/coder/tree/main/examples/templates)
 > directory in the repo.
 
-## Configure Max Workspace Autostop
+## Template policies (enterprise)
 
-To control cost, specify a maximum time to live flag for a template in hours or
-minutes.
+By default, developers have unlimited workspace usage and can use any template version. Template policies allows you to:
```

Comment:
Can't a developer use only the active version of a template? Only admins can create workspaces from any version.

Reply:
We plan on making the behavior configurable per-template.

Reply:
But restricting users to only use the active template is a good double-check if we do something like [this](https://github.com/coder/coder/issues/8216#issuecomment-1608319249)